mbedtls/src/cipher/raw/mod.rs

```diff
@@ -168,13 +205,26 @@ define!(
 define!(
     #[c_ty(cipher_context_t)]
     #[repr(C)]
-    struct Cipher;
-    const init: fn() -> Self = cipher_init;
+    struct Cipher {
+        iv: Option<Vec<u8>>,
```

Comment:
Why is it now necessary to store the IV on the rust object? It is normally stored inside the mbedtls object after `set_key_iv`

Reply:
e.g. it may also be present in `inner`